index.js

```diff
@@ -119,4 +140,10 @@ console.log('hello world')
  * Example Output: 2, 10, 20
  */
 
-// TODO: ADD CODE HERE
+function multiplyByTwo(number) {
+  return number * 2;
+}
+  let num1 = multiplyByTwo(1);
+  let num2 = multiplyByTwo(5);
+  let num3 = multiplyByTwo(10);
```

Comment:
since these lines aren't inside a new code block (anything between the `{}`) they don't need to be indented two spaces.